apps/performance-tests/src/util/Datasets.ts

```diff
@@ -5,10 +5,32 @@
 
 import fs from "fs";
 import path from "path";
-import { insertPhysicalElement, insertPhysicalModelWithPartition, insertSpatialCategory } from "presentation-test-utilities";
+import {
+  insertDrawingCategory,
+  insertDrawingGraphic,
+  insertDrawingModelWithPartition,
+  insertFunctionalElement,
+  insertFunctionalModelWithPartition,
+  insertGroupInformationElement,
+  insertGroupInformationModelWithPartition,
+  insertPhysicalElement,
+  insertPhysicalModelWithPartition,
+  insertSpatialCategory,
+  insertSubCategory,
+  insertSubject,
+} from "presentation-test-utilities";
 import { createIModel } from "./IModelUtilities";
 
-export const IMODEL_NAMES = ["baytown", "50k elements"] as const;
+export const IMODEL_NAMES = [
+  "baytown",
+  "50k elements",
+  "50k nested elements",
+  "50k group member elements",
+  "1k nested subjects",
+  "50k subcategories",
+  "50k nested functional 3D elements",
+  "10k nested functional 2D elements",
```

Comment:
The query was timing out, though it seems that was happening because it was performing a larger query than I intended. Fixed and switched to using 50k 2D elements.